src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/MemoryBlocks/StreamMemoryBlockProvider.cs

```diff
@@ -101,29 +101,29 @@ protected override AbstractMemoryBlock GetMemoryBlockImpl(int start, int size)
 
             if (_useMemoryMap && size > MemoryMapThreshold)
             {
-                MemoryMappedFileBlock block;
+                MemoryMappedFileBlock? block;
```

Comment:
NIT: Normally we move out parameter declaration to the method call, but not necessary, up to you
`if (TryCreateMemoryMappedFileBlock(absoluteStart, size, out MemoryMappedFileBlock? block))`

Reply:
Sure, I generally avoid changes not directly related to the scope of a PR, but this is innocuous enough.